client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1431,6 +1431,8 @@
             },
             templateUrl: 'scripts/superdesk-authoring/views/send-item.html',
             link: function sendItemLink(scope, elem, attrs) {
+                var parentScope = scope.$parent.$parent;
```

Comment:
`Monitoring Controller` is accessible with `require` on monitoring page. But it is not accessible on authoring page. So, as this two pages use same directive for sending item, on monitoring view works fine, but on authoring screen it throws an error. Because of that I added `if (parentScope.monitoring) {` .